### ISSUE TITLE
On Screen Controls Cover Picture

### ISSUE THREAD

**Issue by Addlai** [iina/iina#4158]
**System and IINA version:**

- macOS 
- IINA 

**Expected behavior:**
I expected a standard Mac UI, of course. That means the application would have the standard layout for a Mac application, with a titlebar at the top of the window, and either a toolbar right beneath that, or a dedicated area for controls somewhere else on the window. I would expect these elements to be fixed in their positions, as with any normal application, not appear and disappear, or follow some idiosyncratic behavior. Also, it's natural with a media playback application to want to have a position marker or time indicator to know where you are in the playback.

**Actual behavior:**
The actual behavior of this application is highly non-standard and bizarre. (It is hard for me to know if this a bug or simply a very idiosyncratic behavior.) The titlebar only appears intermittently but always eventually disappears. While it is present, it does not appear to be a normal titlebar, but instead partly covers the video that is playing, which is obviously undesirable, as well as very confusing. It appears as if the titlebar were inside the application area of the window rather than on top of it.

The controls also only appear intermittently and when they do, they cover the picture, which again is obviously undesirable.

If this is intended behavior, it is at the discretion of the developers to choose such behavior. I personally strongly value UI standards and consider deviations from standards to be, almost always, or perhaps always, novice mistakes. I suggest that if the developers choose a highly non-standard UI, they also give the option (say in the preferences) to have a standard UI.

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**

**Comment by saagarjha** [comment]
Have you used QuickTime Player? Do your concerns apply to that application too?

**Comment by ruslaniv** [comment]
Yes, I completely agree.  
On screen controls are completely broken on my mac. 
When I invoke a "playlist and chapters" window in almost 50% of the cases there is no way to close it. I think the intended behaviour is to click inside the app's windows outside of the playlist and chapters window and it will close automatically but this works absolutely randomly. I have to click on playlist invocation button and off of playlist several times to eventually have the playlist close. There is no dedicated close button. 
The same applies to "quick settings window" which also does not work consistently to close it. 

> The controls also only appear intermittently and when they do, they cover the picture, which again is obviously undesirable.

Agree

**Comment by ruslaniv** [comment]
> Have you used QuickTime Player? Do your concerns apply to that application too?

I don't think this is a very good answer. You don't measure yourself to the worst or average, but to the best.

**Comment by veitch123** [comment]
agreed. I'm searching for an app that doesn't put the controls over the picture. Will there be an option for this? Can there be?

**Comment by uiryuu** [comment]
Fixed in #6132.

**Comment by ruslaniv** [comment]
Could you please provide a guide how I can do this since I can't figure it out (I am on version 1.4.3)

**Comment by uiryuu** [comment]
This feature has been partially implemented in the `develop` branch and will be included in the next major release of IINA (v1.5.0). Further improvements, adjustments, and bug fixes related to this issue are expected. If you're interested, you can try it out by downloading a [Nightly build](https://iina.io/nightly/) (a GitHub account is required).
